js/build_map.js

```diff
@@ -91,18 +91,29 @@ var tooltipDiv = d3.select("body").append("div")
   .classed("tooltip hidden", true);
 
 // Read data and add to map
-var dataQueue = d3.queue();
+var stateDataFile;
 if(waterUseViz.interactionMode === 'tap') {
-  dataQueue.defer(d3.json, "data/state_boundaries_mobile.json");
+  stateDataFile = "data/state_boundaries_mobile.json";
 } else {
-  dataQueue.defer(d3.json, "data/state_boundaries_USA.json");
+  stateDataFile = "data/state_boundaries_USA.json";
 }
-dataQueue
-  .defer(d3.tsv, "data/county_centroids_wu.tsv")
-  .defer(d3.json, "data/wu_data_15_range.json")
-  .defer(d3.json, "data/wu_data_15_sum.json")
-  .defer(d3.json, "data/wu_state_data.json")
-  .await(fillMap);
+
+d3.json(stateDataFile, function(error, data1) {
+	if (error) throw error;
+  d3.tsv("data/county_centroids_wu.tsv", function(error, data2) {
+	  if (error) throw error;
+    d3.json("data/wu_data_15_range.json", function(error, data3) {
+	    if (error) throw error;
+      d3.json("data/wu_data_15_sum.json", function(error, data4) {
+	      if (error) throw error;
+        d3.json("data/wu_state_data.json", function(error, data5) {
+	        if (error) throw error;
+          fillMap(data1, data2, data3, data4, data5);
+        });
+      });
+    });
+  });
+});
```

Comment:
very pretty code nest! =)

Reply:
i'm wondering if it would be more readable to assign the read-in data to globals just after these if(error) lines rather than using data1, data2, etc. as placeholders. that way a higher fraction of the variables would have meaningful names, and you wouldn't have to keep track of the order of data file loading within fillMap.

Reply:
so I renamed to something meaningful, but you're saying read as globals? Like put them into waterUseViz?

Reply:
Yeah, I meant put them straight into the globals - see below. But I also don't mean to be a huge pain about this. If you're ready to move on to other tasks, I'm OK with this as in 07609c0.